### PR TITLE
fix: use return value of torch.clamp for weight clamping

### PR DESCRIPTION
**Pull request by Mr-Neutr0n**
## Summary

- `torch.clamp()` is **not** in-place — it returns a new tensor. In both `GE2ELoss.forward()` and `AngleProtoLoss.forward()` in `TTS/encoder/losses.py`, the call `torch.clamp(self.w, 1e-6)` discards the return value, so the weight parameter `self.w` is never actually clamped to a minimum of `1e-6`.
- Replaced with `self.w.data.clamp_(1e-6)` which correctly clamps the parameter's underlying data in-place, ensuring the weight scaling factor stays positive as intended.

## Details

The bug exists in two locations in `TTS/encoder/losses.py`:

**`GE2ELoss.forward()` (line 115):**
```python
# Before (bug): return value discarded, self.w unchanged
torch.clamp(self.w, 1e-6)

# After (fix): in-place clamp on parameter data
self.w.data.clamp_(1e-6)
```

**`AngleProtoLoss.forward()` (line 159):**
```python
# Before (bug): return value discarded, self.w unchanged
torch.clamp(self.w, 1e-6)

# After (fix): in-place clamp on parameter data
self.w.data.clamp_(1e-6)
```

Without this fix, if `self.w` becomes negative during training (via gradient updates), the cosine similarity matrix scaling would be inverted, potentially destabilizing speaker encoder training.

## Test plan

- [ ] Verify `self.w` stays non-negative during GE2E loss training
- [ ] Verify `self.w` stays non-negative during AngleProto loss training
- [ ] Run existing encoder loss unit tests

### CODE REVIEW

**Comment by Mr-Neutr0n** [comment]
closing this out — saw the note about the repo being unmaintained. will redirect to the active fork if still relevant
